tendrl/commons/persistence/etcd_persister.py

```diff
@@ -1,14 +1,8 @@
-from tendrl.commons.etcdobj.etcdobj import Server as etcd_server
 from tendrl.commons.persistence.persister import Persister
 
 
 class EtcdPersister(Persister):
-    def __init(self, config):
-        super(EtcdPersister, self).__init__(config)
```

Comment:
I feel passing config was more flexible option and later if we node something more from config, we would have to change the signature of constructor again to pass additional data.